pandas/core/frame.py

```diff
@@ -1655,8 +1655,8 @@ def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
             defaults to 'ascii' on Python 2 and 'utf-8' on Python 3.
         compression : string, optional
             a string representing the compression to use in the output file,
-            allowed values are 'gzip', 'bz2', 'xz',
-            only used when the first argument is a filename
+            allowed values are 'gzip', 'bz2', 'zip', 'xz', only used when the
+            first argument is a filename.
```

Comment:
Let's fix this parameter description a bit:
~~~
a string representing the compression to use in the output file.
Allow values are 'gzip', 'bz2', 'zip', 'xz'.  This input is only used
when the first argument is a filename.
~~~

Reply:
done.